site/params.json

```diff
@@ -1 +1 @@
-{"name":"Weave","tagline":"The Docker Network","body":"","google":"","note":"Don't delete this file! It's used internally to help with page regeneration."}
+{"name":"Weave","tagline":"weaveing containers into applications","body":"","google":"","note":"Don't delete this file! It's used internally to help with page regeneration."}
```

Comment:
typo: "weaveing"

(also, isn't this generated by github pages -- is it OK to change it?)